db.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function(pg, prettify){
+	var conectionString = process.env.DATABASE_URL || "postgres://postgres:123456@localhost/uvs";
+
+	function query(sql, params, callback) {
+	    pg.connect(conectionString, function(err, client, done) {
+	    	if (err) {
+	    		callback(err)
+	        	return console.error('error fetching client from pool', err);
+	     	}
+	    	client.query(sql, params, function(err, result) {
+	        	done();
+	        	if (err) {
+	        		console.error('error running query', err);
+	        	}
+	        	if (callback) {
+	        		result.rows =  prettify(result.rows) // e
```

Comment:
ok, show me the code we have that will not work or will look ambiguous without this?